### PR TITLE
Add new content

### PR DESCRIPTION
**Pull request by InessaPawson**
Adding new content.

### CODE REVIEW

File: content/en/citing-numpy.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+**Citing NumPy**
```
Comment (rgommers) [inline content/en/citing-numpy.md]: To create a page title, the format should be:
```
---
title: Citing NumPy
sidebar: false
---
```

File: content/en/citing-numpy.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+If NumPy has been significant in your research, and you would like to acknowledge the project in your academic publication, we suggest citing the following papers:
+
+
+
+*   Travis E, Oliphant. A guide to NumPy, USA: Trelgol Publishing, (2006).
+*   Stéfan van der Walt, S. Chris Colbert and Gaël Varoquaux. The NumPy Array: A Structure for Efficient Numerical Computation, Computing in Science & Engineering, 13, 22-30 (2011),[ DOI:10.1109/MCSE.2011.37](http://dx.doi.org/10.1109/MCSE.2011.37) ([publisher link](http://scitation.aip.org/content/aip/journal/cise/13/2/10.1109/MCSE.2011.37))
+
+_In BibTeX format:_
+
```
Comment (rgommers) [inline content/en/citing-numpy.md]: This is missing the first line: `@article{van2011numpy,`

It also doesn't render well like this. Instead try to use triple backticks above and below this:
```
@article{van2011numpy,
  title={The NumPy array: a structure for efficient numerical computation},
  author={Van Der Walt, Stefan and Colbert, S Chris and Varoquaux, Gael},
  journal={Computing in Science \& Engineering},
  volume={13},
  number={2},
  pages={22},
  year={2011},
  publisher={IEEE Computer Society}
}
```

File: content/en/numpy-code-of-conduct.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
+<h2>NumPy Code of Conduct</h2>
```
Comment (rgommers) [inline content/en/numpy-code-of-conduct.md]: Same comment as above, use:
```
---
title: NumPy Code of Conduct
sidebar: false
---

```

Comment (rgommers) [reply content/en/numpy-code-of-conduct.md]: Also, can you rename the file to simply `code-of-conduct.md`? Looks better, and will make the link in the footer work.

File: content/en/numpy-code-of-conduct.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+<h3>Introduction</h3>
```
Comment (rgommers) [inline content/en/numpy-code-of-conduct.md]: Use Markdown here, so for a `h3` it's `### Introduction`

File: content/en/citing-numpy.md (hunk 1 above)
Comment (InessaPawson) [reply content/en/citing-numpy.md]: Thank you, @rgommers ! I'll keep it in mind next time.

File: content/en/numpy-code-of-conduct.md (hunk 4 above)
Comment (InessaPawson) [reply content/en/numpy-code-of-conduct.md]: @rgommers Thank you for all your advice! 

For the Terms of Use page I used termly.io/products/terms-and-conditions-generator/ to fit the needs of numpy.org.

Re: privacy.md My vote goes to the second option - referencing and linking to https://numfocus.org/privacy-policy. I’ll change the text on the Privacy page accordingly.

File: content/en/numpy-code-of-conduct.md (hunk 3 above)
Comment (InessaPawson) [reply content/en/numpy-code-of-conduct.md]: Done.

File: content/en/numpy-code-of-conduct.md (hunk 4 above)
Comment (InessaPawson) [reply content/en/numpy-code-of-conduct.md]: > Use Markdown here, so for a `h3` it's `### Introduction`

Done.

**Comment by rgommers** [comment]
Privacy page linking, citing and CoC look good to me now.

> For the terms of use page, can you comment on where the content came from?

Still have this question. I'd rather avoid the legal stuff if at all possible. I think NumFOCUS would want to run that by a lawyer if we keep it like that.

**Comment by rgommers** [comment]
Ah, I missed the answer to my terms-of-use question, it was in a resolved comment:

> For the Terms of Use page I used termly.io/products/terms-and-conditions-generator/ to fit the needs of numpy.org.
>
> Re: privacy.md My vote goes to the second option - referencing and linking to https://numfocus.org/privacy-policy. I’ll change the text on the Privacy page accordingly.

And some more feedback I got from @InessaPawson on these terms and conditions:

> I used termly.io/products/terms-and-conditions-generator/ to fit the needs of numpy.org.
>
> This is a good article explaining why every website needs the Terms of Use page: termsfeed.com/blog/5-reasons-need-terms-conditions/

Good arguments, I'll review in detail and merge if I don't have specific concerns.

**Comment by rgommers** [comment]
LGTM now, pushed a few fixes and merged. Thanks Inessa!

**Pull request merged**